ngx-tools/testing/src/mocks/window.service.mock.ts

```diff
@@ -22,6 +22,8 @@ const windowMock: Window = {
       addRange: noop,
     };
   },
+  scrollTo: (x: number, y: number) => {
+  },
```

Comment:
that arrow function looks a little too pointy. not sure this should be approved...